pkg/config/proxy.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+// ProxyConfig specifies the properties required to run the proxy
+type ProxyConfig struct {
+	StorageType           string `validate:"required" envconfig:"ATHENS_STORAGE_TYPE" default:"mongo"`
+	OlympusGlobalEndpoint string `validate:"required" envconfig:"OLYMPUS_GLOBAL_ENDPOINT" default:"olympus.gomods.io"`
```

Comment:
i think this changed in some recent PR to localhost

Reply:
didn't realize, fixed!